contrib/playlib/src/mill/playlib/RouteCompilerWorkerApi.scala

```diff
@@ -20,7 +20,7 @@ private[playlib] class RouteCompilerWorker {
         ctx.log.debug("Loading classes from\n"+toolsClassPath.mkString("\n"))
         val cl = mill.api.ClassLoader.create(
           toolsClassPath,
-          null,
```

Comment:
This won't work and is most likely the cause of the test failure.  `null` as parent classloader is needed to separate the potential different binary-incompatible Scala versions of mill and the route compiler.